lib/customer.rb

```diff
@@ -0,0 +1,65 @@
+require_relative 'restaurant'
+
+class Customer
+  attr_accessor :order
+
+  def initialize
+    @order = []
+  end
+
+  def see_menu(restaurant)
+    output = ""
+    restaurant.menu.each do |dish, price|
+      output += "#{dish} are £#{price}\n"
+    end
+    puts output
+    output
+  end
+
+  def get_order(restaurant)
+    see_menu(restaurant)
+    current_order = []
+    puts "Enter item:"
+    item = gets.chomp
+    puts "Enter amount:"
+    num = gets.chomp.to_i
+    current_order << item
+    current_order << num
+    check_order(current_order, restaurant)
+    puts "Total cost of order is £#{total_price(restaurant)}"
+  end
+
+  def check_order(current_order, restaurant)
+    if restaurant.menu.include?(current_order[0]) && current_order[1].positive? && current_order[1].integer?
```

Comment:
This line is hard to understand on its own. You could have `check_order` take three parameters so that the naming is clearer.